Header header
string[] x_points
string[] y_points
string[] z_points
